subsys/bluetooth/audio/tbs_client.c

```diff
@@ -0,0 +1,2166 @@
+static uint8_t read_technology_cb(struct bt_conn *conn, uint8_t err,
+				   struct bt_gatt_read_params *params,
+				   const void *data, uint16_t length)
+{
+	struct bt_tbs_instance *inst = CONTAINER_OF(params, struct bt_tbs_instance, read_params);
+
+	(void)memset(params, 0, sizeof(*params));
+
+	if (inst != NULL) {
+		uint8_t cb_err = err;
+		uint8_t technology = 0;
+
+		if (IS_ENABLED(CONFIG_BT_TBS_CLIENT_GTBS) && inst->gtbs) {
+			BT_DBG("GTBS");
+		} else {
+			BT_DBG("Index %u", inst->index);
+		}
+
+		if (err != 0) {
+			BT_DBG("err: 0x%02X", err);
+		} else if (data != NULL) {
+			BT_HEXDUMP_DBG(data, length, "Data read");
+			if (length == sizeof(technology)) {
+				(void)memcpy(&technology, data, length);
+				BT_DBG("%s (0x%02x)",
+				       bt_tbs_technology_str(technology),
+				       technology);
+			} else {
+				BT_DBG("Invalid length");
+				cb_err = BT_ATT_ERR_INVALID_ATTRIBUTE_LEN;
+			}
+		}
+		inst->busy = false;
+
+		if (tbs_client_cbs != NULL && tbs_client_cbs->technology != NULL) {
+			tbs_client_cbs->technology(conn, cb_err, inst->index,
+					    technology);
+		}
+	}
+
+	return BT_GATT_ITER_STOP;
+}
+
+static uint8_t read_uri_list_cb(struct bt_conn *conn, uint8_t err,
+				    struct bt_gatt_read_params *params,
+				    const void *data, uint16_t length)
+{
+	struct bt_tbs_instance *inst = CONTAINER_OF(params, struct bt_tbs_instance, read_params);
+
+	(void)memset(params, 0, sizeof(*params));
+
+	if (inst != NULL) {
+		const char *uri_scheme_list = NULL;
+
+		if (IS_ENABLED(CONFIG_BT_TBS_CLIENT_GTBS) && inst->gtbs) {
+			BT_DBG("GTBS");
+		} else {
+			BT_DBG("Index %u", inst->index);
+		}
+
+		if (err != 0) {
+			BT_DBG("err: 0x%02X", err);
+		} else if (data != NULL) {
+			uri_scheme_list = parse_string_value(data, length,
+						MAX_URI_SCHEME_LIST_SIZE);
+			BT_DBG("%s", log_strdup(uri_scheme_list));
+		}
+		inst->busy = false;
+
+		if (tbs_client_cbs != NULL && tbs_client_cbs->uri_list != NULL) {
+			tbs_client_cbs->uri_list(conn, err, inst->index,
+					  uri_scheme_list);
+		}
+	}
+
+	return BT_GATT_ITER_STOP;
+}
+
+static uint8_t read_signal_strength_cb(struct bt_conn *conn, uint8_t err,
+					struct bt_gatt_read_params *params,
+					const void *data, uint16_t length)
+{
+	struct bt_tbs_instance *inst = CONTAINER_OF(params, struct bt_tbs_instance, read_params);
+
+	(void)memset(params, 0, sizeof(*params));
+
+	if (inst != NULL) {
+		uint8_t cb_err = err;
+		uint8_t signal_strength = 0;
+
+		if (IS_ENABLED(CONFIG_BT_TBS_CLIENT_GTBS) && inst->gtbs) {
+			BT_DBG("GTBS");
+		} else {
+			BT_DBG("Index %u", inst->index);
+		}
+
+		if (err != 0) {
+			BT_DBG("err: 0x%02X", err);
+		} else if (data != NULL) {
+			BT_HEXDUMP_DBG(data, length, "Data read");
+			if (length == sizeof(signal_strength)) {
+				(void)memcpy(&signal_strength, data, length);
+				BT_DBG("0x%02x", signal_strength);
+			} else {
+				BT_DBG("Invalid length");
+				cb_err = BT_ATT_ERR_INVALID_ATTRIBUTE_LEN;
+			}
+		}
+		inst->busy = false;
```

Comment:
Same here. Won't comment in other places if I see it again.

Reply:
Fixed it everywhere. 

This is one of the rules that I referenced in https://github.com/zephyrproject-rtos/zephyr/pull/40707#discussion_r761788085 that are unwritten and is only enforced by a few people. 

If we truly want to enforce such specific rules for the code base, those should be checked for in CI, or at least be written down in the official coding guidelines.

Reply:
I suspect it's going to be seen as a too minor thing to be enforced tree-wide.

Reply:
I also think that while tree-wide consistency is important, the importance of consistency is inversely proportional to the size of the scope that's being viewed (e.g. it's more important for code within a single c-file to be consistent than two c-files from different subsystems in the same tree).